apis/v1alpha2/backendtlspolicy_types.go

```diff
@@ -25,7 +25,6 @@ import (
 // +genclient
 // +kubebuilder:object:root=true
 // +kubebuilder:subresource:status
-// +kubebuilder:storageversion
```

Comment:
I've spent a fair amount of time thinking about our least bad option here, and I think we should entirely drop BackendTLSPolicy v1alpha2 from this release. Unfortunately this will result in a painful migration for anyone that has already installed BackendTLSPolicy in their clusters. They'll need to uninstall the v1alpha2 CRD and then install the v1alpha3 CRD since there will be no way to upgrade in place. My rationale for this approach is:

1. This avoids surprises. It's impossible to accidentally install the newer CRD and break your controller. 
2. It would be very difficult for a controller to try to support both API versions simultaneously and that seems unlikely. Instead users will likely need to coordinate upgrading the CRD alongside their controller of choice.
3. Any version of us supporting both API versions simultaneously would rely on either us deploying a conversion webhook (something we've consistently and intentionally avoided) or configuring this CRD to preserve unknown fields + suggesting controllers use dynamic client, but both of those options are filled with danger and do not seem to be worth the risk or effort.
4. We've said that [experimental channel can have breaking changes](https://gateway-api.sigs.k8s.io/concepts/versioning/#what-can-change), if we never actually break anything, people will begin to expect a higher level of stability than we've intended here.

I've run these ideas by @kflynn and @shaneutt and they both seem to be on board, but would really appreciate any additional feedback on this idea before we go ahead.

/cc @arkodg @LiorLieberman @mikemorris @mlavacca

Reply:
I'd like to cc @skriss @sunjayBhatia for feedback, since Contour has released an implementation of BackendTLSPolicy v1a2.  https://github.com/projectcontour/contour/blob/main/changelogs/CHANGELOG-v1.28.0.md

Reply:
Done.  Contour folks are aware of the upcoming change.